setup.py

```diff
@@ -0,0 +1,36 @@
+from setuptools import find_packages, setup
+
+
+setup(
+    name="blackbox",
+    version="0.0.1",
+    description="Tool for automatic backups of databases",
+    long_description=open("README.md").read(),
+    long_description_content_type="text/markdown",
+    author="Leon Sandoy",
+    author_email="heavysaturn@gmail.com",
```

Comment:
```suggestion
    author_email="leon.sandoy@gmail.com",
```
Who the fuck is heavysaturn@gmail.com